bin/Powheg/run_pwg.py

```diff
@@ -968,10 +982,13 @@ def runhnnlo(folderName, njobs, QUEUE):
         os.system('mkdir -p '+rootfolder+'/'+args.folderName)
         os.system('cp -p '+args.inputTemplate.split('/')[-1]+' '+args.folderName+'/powheg.input')
 
-        os.system('rm -rf JHUGen.input')
-        inputJHUGen = '/'.join(powInputName.split('/')[0:-1])+'/JHUGen.input'
+        #os.system('rm -rf JHUGen.input')
+        #inputJHUGen = '/'.join(powInputName.split('/')[0:-1])+'/JHUGen.input'
```

Comment:
It should be done now.
Let me know if you are able to merge now.
Thanks!
Giulio